test/new-e2e/tests/installer/systemd.go

```diff
@@ -1,90 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
-
-// Package updater contains tests for the updater package
-package updater
-
-import (
-	"encoding/json"
-	"sort"
-	"strings"
-	"testing"
-	"time"
-
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func getMonotonicTimestamp(t *testing.T, host *components.RemoteHost) int64 {
-	res := strings.TrimSpace(host.MustExecute("sudo journalctl -n 1 --output=json 2> /dev/null"))
-	var log JournaldLog
-	err := json.Unmarshal([]byte(res), &log)
-	require.NoError(t, err, res)
-	return log.MonotonicTimestamp
-}
-
-// JournaldLog represents a log entry from journald
```

Comment:
why do you remove the experiment test `TestExperimentCrash` ? and those helpers to test experiments

Reply:
backported them